utils/glam_subdags/generate_query.py

```diff
@@ -150,7 +150,7 @@ def generate_and_run_glean_task(
         "DATASET": destination_dataset_id,
         "SUBMISSION_DATE": "{{ ds }}",
         "IMPORT": "true",
-        "USE_SAMPLE_ID": use_sample_id,
+        "USE_SAMPLE_ID": str(use_sample_id),
```

Comment:
Does it matter that this will be `True` or `False`, while the above `IMPORT` env var is lowercase `true`?

Reply:
Not really. [On the other side](https://github.com/mozilla/bigquery-etl/pull/6155/files#) the code is ready to receive `True` or `False`

Reply:
While I ack that the difference in casing is not the best, though. Not sure if that's what you're referring to.